docs/topics/wasm/wasm-debugging.md

```diff
@@ -152,8 +119,12 @@ To use this feature, ensure that custom formatters are enabled in your browser's
 
   ![Enable custom formatters in Firefox](wasm-custom-formatters-firefox.png){width=400}
 
-Custom formatters work for Kotlin/Wasm development builds. If you have specific requirements for production builds,
-you need to adjust your Gradle configuration accordingly. Add the following compiler option to the `wasmJs {}` block:
+This feature is supported in Firefox and Chromium-based browsers as
+it uses the [custom formatters API](https://firefox-source-docs.mozilla.org/devtools-user/custom_formatters/index.html).
```

Comment:
This reads like "this feature is supported _only_ in Firefox and Chromium". If that's not the case, I'd change the order: "This feature uses the custom formatters API, and is supported in Firefox and Chromium".